src/controllers/searchResult/base.js

```diff
@@ -45,23 +45,24 @@ class SearchResultBase {
           return this;
         });
     }
-    else if (this._request.size && this._request.sort) {
+    else if (this._request.size && this._request.body.sort) {
       const
         request = Object.assign({}, this._request, {
-          action: this._searchAction,
-          search_after: []
+          action: this._searchAction
         }),
-        hit = this._response.hits && this._response.hits[this._response.hits.length -1];
+        hit = this._response.hits[this._response.hits.length -1];
 
-      for (const sort of this._request.sort) {
+      request.body.search_after = [];
+
+      for (const sort of this._request.body.sort) {
         const key = typeof sort === 'string'
           ? sort
           : Object.keys(sort)[0];
         const value = key === '_uid'
```

Comment:
(nitpicking) maybe adding parentheses make it more readable
```suggestion
        const value = (key === '_uid')
```